src/modules/collection/reducer.ts

```diff
@@ -187,7 +197,8 @@ export function collectionReducer(state: CollectionState = INITIAL_STATE, action
               }
             }
           : {}),
-        error: null
+        error: null,
+        lastFetchParams: params
```

Comment:
Would you mind adding a test / changing tests to ensure that this variable is set when the success action is called?